### ISSUE TITLE
Benediktenwand - Topo and path do not correlate

### ISSUE THREAD

**Issue by fire-burner** [der-stefan/OpenTopoMap#244]
First of all thanks for the great project!
I found a small issue on a Souther German mountain path on Benediktenwand:
When coming from the West flank, the path does not go up for more than 30m before reaching the Tutzinger Hut.
see https://opentopomap.org/#map=17/47.65590/11.46023
As this is my first report, I am not sure if this is the correct way to address this issue.

**Comment by max-dn** [comment]
Yes, the path sees to correct, the elevation data are wrong (with better data you get[ a small increase with 7m](https://geo.dianacht.de/topo/?track=39439) height at this point). That's a pity, but I'm afraid we don't have any better ones for use in OpenTopoMap. See #191

**Comment by fire-burner** [comment]
I see, so the issue is with the Topo data not always being that accurate.
Thanks for the clarification. I guess it would be great if the state would open source their topography maps eventually.

**Comment by der-stefan** [comment]
Closing for now. Next DEM will be Copernicus 30m (COP-30).
